src/com/goide/GoDocumentationProvider.java

```diff
@@ -85,7 +85,7 @@ else if (element instanceof GoTypeSpec) {
   }
 
   @NotNull
-  private static List<PsiComment> getCommentsInner(@Nullable PsiElement element) {
+  public static List<PsiComment> getCommentsInner(@Nullable PsiElement element) {
```

Comment:
It even contains `inner` word in name, it means that nobody should build its logic depending on this method